rest-server/src/controllers/job.js

```diff
@@ -25,28 +26,16 @@ const logger = require('../config/logger');
 const load = (req, res, next, jobName) => {
   new Job(jobName, (job, error) => {
     if (error) {
-      if (error.message === 'JobNotFound') {
+      if (error.code === 'ERR_NO_JOB') {
         if (req.method !== 'PUT') {
-          logger.warn('load job %s error, could not find job', jobName);
-          return res.status(404).json({
-            error: 'JobNotFound',
-            message: `could not find job ${jobName}`,
-          });
+          return next(error);
```

Comment:
error --> createError(...) ?

Reply:
Since this error has `code` property in this branch, assume that it has been already transformed into `HttpError` in `utils/error.js`